next-tavla/src/Admin/scenarios/TileSettings/components/QuaySettings.tsx

```diff
@@ -0,0 +1,14 @@
+import { SelectLines } from 'Admin/scenarios/SelectLines'
+import { GetQuayQuery } from 'graphql/index'
+import { useQuery } from 'graphql/utils'
+import { TQuayTile } from 'types/tile'
+import { fieldsNotNull } from 'utils/typeguards'
+
+function QuaySettings({ tile }: { tile: TQuayTile }) {
+    const { data } = useQuery(GetQuayQuery, { quayId: tile.placeId })
+
+    const lines = data?.quay?.lines.filter(fieldsNotNull) ?? []
+
+    return <SelectLines tile={tile} lines={lines} />
```

Comment:
![image](https://github.com/entur/tavla/assets/18345134/5e69f142-cc29-4b12-910b-4d60825f4572)
Er noe litt rart med hvordan selectlines blir seende ut

Reply:
Mener du "velg linjer"- expandable eller listen? Jeg har ikke gjort noe med den både siden det ikke er scope for oppgaven, men også fordi jeg antar expandable skal fjernes i følge designet